.travis.yml

```diff
@@ -20,9 +21,9 @@ matrix:
     - php: 5.5
       env: SYMFONY_VERSION=2.6.*
     - php: 5.5
-      env: SYMFONY_VERSION=2.7.*
+      env: SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=full
     - php: 5.5
-      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=full
```

Comment:
there should be no deprecation warnings with symfony 2.7 or 2.8. the constant should be anything but weak to have deprecations reported as errors.